homeworks/hw25/conftest.py

```diff
@@ -0,0 +1,14 @@
+# pylint: disable=import-error,duplicate-code
+import pytest
+from playwright.sync_api import sync_playwright
+
+
+@pytest.fixture(scope='class')
+def browser_page():
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=False)
+        context = browser.new_context()
+        page = context.new_page()
+        page.goto('https://www.saucedemo.com/')
```

Comment:
fixture should just start/open browser, it shouldn’t depend on the site, so move this `page.goto('https://www.saucedemo.com/')` out